client.go

```diff
@@ -180,13 +179,24 @@ type ovndb struct {
 	tranmutex    sync.Mutex
 	signalCB     OVNSignal
 	disconnectCB OVNDisconnectedCallback
+	db           string
 }
 
-func NewClient(cfg *Config, db string) (Client, error) {
+func NewClient(cfg *Config) (Client, error) {
+	db := cfg.db
+	// db string should strictly be OVN_Northbound or OVN_Southbound
+	if db == "" {
+		// default to OVN_Northbound
+		db = dbNB
+	} else if !(db == dbNB || db == dbSB) {
+		return nil, fmt.Errorf("db %s is not supported option", db)
```

Comment:
better to make it clear in the error message: the valid "db" names are: "OVN_Northbound", "OVN_Southbound"

Reply:
done